### DISCUSSION TITLE
Can't select preferred admin

### DISCUSSION THREAD

**Discussion by LazeX7** [CyferShepard/Jellystat#210] [Q&A]
Hello,

I have installed Jellystat and I can't get it to work. The only thing I can't do is to select a admin.
When I click on " Select a preferred Admin" You can see a thin bar appearing, i'm assuming that should be the dropdown menu.

But I can't see nothing and can't select anything.
![image](https://github.com/CyferShepard/Jellystat/assets/14087562/5fb49f9e-3614-45ea-93c4-e0f662938935)

**Comment by LazeX7** [comment]
Nobody can answer this?

**Comment by CyferShepard** [comment]
Hey @LazeX7 , sorry, i meant to reply but got busy during the week, can you check under `Library Settings` tab next to this if any of your libraries are visible? if not, can you check the docker logs for any errors. if your libraries are showing, can you confirm if you have a valid admin account thats active?

**Reply by LazeX7** [reply]
Hey!

Don't worry! I don't see anything under library settings so I checked the docker logs:

ns
 
 
 
      
    { TraceIndex: 6, line: 'node:_http_client:502:9' },
    { TraceIndex: 7, line: 'node:events:520:28' },
    { TraceIndex: 8, line: 'node:internal/streams/destroy:170:8' },
    { TraceIndex: 9, line: 'node:internal/streams/destroy:129:3' },
    { TraceIndex: 10, line: 'node:internal/process/task_queues:82:21' },
    {
      TraceIndex: 11,
      line: '/app/node_modules/axios/dist/node/axios.cjs:3877:41'
    },
    { TraceIndex: 12, line: 'node:internal/process/task_queues:95:5' },
    {
      TraceIndex: 13,
      line: '/app/backend/classes/jellyfin-api.js:358:24'
    },
    {
      TraceIndex: 14,
      line: '/app/backend/tasks/ActivityMonitor.js:24:30'
    }
  ],
  Message: 'connect ECONNREFUSED 127.0.0.1:80',
  url: undefined
}
[JELLYFIN-API] {
  ErrorAt: 'at AxiosError.from (/app/node_modules/axios/dist/node/axios.cjs:837:14',
  ErrorLines: [
    {
      TraceIndex: 1,
      line: '/app/node_modules/axios/dist/node/axios.cjs:837:14'
    },
    {
      TraceIndex: 2,
      line: '/app/node_modules/axios/dist/node/axios.cjs:3087:25'
    },
    { TraceIndex: 3, line: 'node:events:520:28' },
    {
      TraceIndex: 4,
      line: '/app/node_modules/follow-redirects/index.js:38:24'
    },
    { TraceIndex: 5, line: 'node:events:520:28' },
    { TraceIndex: 6, line: 'node:_http_client:502:9' },
    { TraceIndex: 7, line: 'node:events:520:28' },
    { TraceIndex: 8, line: 'node:internal/streams/destroy:170:8' },
    { TraceIndex: 9, line: 'node:internal/streams/destroy:129:3' },
    { TraceIndex: 10, line: 'node:internal/process/task_queues:82:21' },
    {
      TraceIndex: 11,
      line: '/app/node_modules/axios/dist/node/axios.cjs:3877:41'
    },
    { TraceIndex: 12, line: 'node:internal/process/task_queues:95:5' },
    {
      TraceIndex: 13,
      line: '/app/backend/classes/jellyfin-api.js:358:24'
    },
    {
      TraceIndex: 14,
      line: '/app/backend/tasks/ActivityMonitor.js:24:30'
    }
  ],
  Message: 'connect ECONNREFUSED 127.0.0.1:80',
  url: undefined
}
[JELLYFIN-API] {
  ErrorAt: 'at AxiosError.from (/app/node_modules/axios/dist/node/axios.cjs:837:14',
  ErrorLines: [
    {
      TraceIndex: 1,
      line: '/app/node_modules/axios/dist/node/axios.cjs:837:14'
    },
    {
      TraceIndex: 2,
      line: '/app/node_modules/axios/dist/node/axios.cjs:3087:25'
    },
    { TraceIndex: 3, line: 'node:events:520:28' },
    {
      TraceIndex: 4,
      line: '/app/node_modules/follow-redirects/index.js:38:24'
    },
    { TraceIndex: 5, line: 'node:events:520:28' },
    { TraceIndex: 6, line: 'node:_http_client:502:9' },
    { TraceIndex: 7, line: 'node:events:520:28' },
    { TraceIndex: 8, line: 'node:internal/streams/destroy:170:8' },
    { TraceIndex: 9, line: 'node:internal/streams/destroy:129:3' },
    { TraceIndex: 10, line: 'node:internal/process/task_queues:82:21' },
    {
      TraceIndex: 11,
      line: '/app/node_modules/axios/dist/node/axios.cjs:3877:41'
    },
    { TraceIndex: 12, line: 'node:internal/process/task_queues:95:5' },
    {
      TraceIndex: 13,
      line: '/app/backend/classes/jellyfin-api.js:358:24'
    },
    {
      TraceIndex: 14,
      line: '/app/backend/tasks/ActivityMonitor.js:24:30'
    }
  ],
  Message: 'connect ECONNREFUSED 127.0.0.1:80',
  url: undefined
}

**Reply by LazeX7** [reply]
I got it fixed, couldn't wait for a reply so I googled on the error messages. One was saying that you have to add http:// in the URL and not just only the IP. now it works.
